tools/snitch/Cargo.lock

```diff
@@ -560,6 +560,7 @@ dependencies = [
  "serde_json",
  "sysinfo",
  "tokio",
+ "tokio-threadpool",
```

Comment:
This is needed now?

Reply:
Yes, due to the notary module spawning a blocking task. `edgelet-docker` previously used `tokio-process`, but we had to remove that during the `crossbeam-utils` upgrade. Relevant discussion: https://github.com/Azure/iotedge/pull/6136#discussion_r809703653.